templates/home/index.html.twig

```diff
@@ -4,4 +4,109 @@
 
 {% block body %}
     <div class="nfq-academy">{{ someVariable }}</div>
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% for key, value in data %}
+            {% for student in value.students %}
+                <li class="list-group-item">
+                    <a href="/student?name={{ student }}&project={{ key|url_encode }}">{{ student }}</a>
```

Comment:
Šaunu, kad naudoti `url_encode`, bet verta žinoti, kad su `path` tą patį galima padaryti paprasčiau/trumpiau. Pvz.:

```twig
{{ path('student', {'name': name, 'otherParameter': other.data }) }}
```

Plačiau:
  * https://symfony.com/doc/current/templating.html#linking-to-pages
  * https://symfony.com/doc/current/reference/twig_reference.html#path